mobile/src/bundles/common/components/modal/styles.ts

```diff
@@ -0,0 +1,17 @@
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+    backdrop: {
+        backgroundColor: 'rgba(0,0,0,0.5)',
+    },
+    container: {
+        width: '80%',
+        gap: 20,
+        backgroundColor: '#fff',
+    },
+    content: {
+        gap: 10,
+    },
+});
```

Comment:
Let's create these 2 components in one folder cause they are similar and have same styles 
```
--common
    --components
       --modals
           --moldal.tsx
           --action-moldal.tsx
           --styles.ts
```